src/firebase-example.js

```diff
@@ -0,0 +1,14 @@
+import firebase from 'firebase/app';
```

Comment:
You don't necessarily have to not include the `firebase.js` in the repo, you could include this file but just store some of the info parts in a `.env` 👍